### PR TITLE
Account map

### PR DESCRIPTION
**Pull request by RCCodeBase**
- Account mapping with names

### CODE REVIEW

**Comment by NiranjanAP** [comment]
@RCCodeBase  couple of automated PR tests are failing, not sure if they are correct, can you check and comment?

**Comment by NiranjanAP** [comment]
@Prashant-Mutnale  can you do the first round of review, so Amar can just focus on the higher level review? that way we can free up Amar for other tasks

**Comment by RCCodeBase** [comment]
@NiranjanAP checked it automated PR tests are reporting lint issues in files that I haven't modified. Also test failing is not related to this file i think.

File: packages/react-components/src/AccountName.tsx [hunk 1]
```diff
@@ -195,11 +196,49 @@ function AccountName ({ children, className = '', defaultName, label, onClick, o
...
+            );
+          } else {
+            setName(() =>
+              identity?.display
+                ? extractIdentity(cacheAddr, identity)
+                : extractName(cacheAddr, accountIndex)
+            );
+          }
+        } else {
+          fetch('https://trust.demo.cord.network/api/v1/mapping/getDetails/' + cacheAddr, {
```
Comment (amarts) [inline packages/react-components/src/AccountName.tsx:215]: Can this URL (the 'https://trust.demo.cord.network) be coming from ENV instead of hardcoding?

**Review by amarts — APPROVED** [review]

**Comment by amarts** [comment]
> @RCCodeBase couple of automated PR tests are failing, not sure if they are correct, can you check and comment?

Checked the failure messages, and looks like they were from other places. There should be a separate PR for fixing them. The changes added in this PR didn't introduce any of them.

**Pull request merged**